helm/sematic-server/values.yaml

```diff
@@ -15,6 +15,10 @@ slack:
 secret:
   create: true
 
+cleaner:
+  crontab: "0-55/5 * * * *"
+  ttlSecondsAfterFinished: 300
```

Comment:
If the frequency is 5 minutes, shouldn't this configuration be well under 5 minutes?

Reply:
no, that's how long the pod object should be kept around after it finishes. Note that by the time the pod is "finished", nothing is *executing* anymore, it's just about how long k8s keeps around the metadata for the pod. And it's perfectly fine for metadata for two pods from the same cron job to be around at once; each pod gets a unique name.